src/MultiDocumenter.jl

```diff
@@ -140,7 +140,26 @@ function make(
     canonical_domain::Union{AbstractString,Nothing} = nothing,
     sitemap::Bool = false,
     sitemap_filename::AbstractString = "sitemap.xml",
+    # This keyword is for internal test use only:
+    _override_windows_isinteractive_check::Bool = false,
 )
+    if Sys.iswindows() && !isinteractive()
+        if _override_windows_isinteractive_check || isinteractive()
+            @warn """
+            Running a MultiDocumenter build interactively in Windows.
+            This should only be used for development and testing, as it will lead to partial
+            and broken builds. See https://github.com/JuliaComputing/MultiDocumenter.jl/issues/70
+            """
+        else
+            msg = """
+            MultiDocumenter deployments are disabled on Windows due to difficulties
+            with handling symlinks in documentation sources.
+            You _can_ test this build locally by running it interactively (i.e. in the REPL).
+            See also: https://github.com/JuliaComputing/MultiDocumenter.jl/issues/70
+            """
+            error(msg)
+        end
+    end
```

Comment:
@pfitzseb So initially I thought we could check if we run into problematic symlinks, and only then error. But I don't think that works, because by default Git on Windows will not create symlinks as far as I can tell. So the "symlinks" are just normal files, and so we can't really distinguish them easily. So instead I added the `isinteractive()` checks here at the very top of `make.jl`.

One consequence here is that we fully "break" running MultiDocumenter on Windows. So we may want to bump this to the next 0.x minor?